packages/story-editor/src/components/panels/design/videoSegment/videoSegment.js

```diff
@@ -62,6 +62,8 @@ const HelperText = styled(Text).attrs({
 `;
 
 function VideoSegmentPanel({ pushUpdate, selectedElements }) {
+  const MIN_SEGMENT_LENGTH = 20;
+  const MAX_SEGMENT_LENGTH = 120;
```

Comment:
We recommend videos that are less than 15 seconds per page, or 60 seconds maximum.

Let's reflect that here accordingly.

```suggestion
  const MIN_SEGMENT_LENGTH = 10;
  const MAX_SEGMENT_LENGTH = 60;
```